rules.ini

```
@@ -314,6 +314,7 @@ Vivox[] = vivox
 Vorbis = vorbis
 Wwise = (?:^|/)AkSoundEngine(?:dll)?\.(?:bundle|dll)$
 XAudio2 = (?:^|/)xaudio2_9redist\.dll$
+YarnSpinner = (?:^|/)[Yy]arn[Ss]pinner(?:\.[Cc]ompiler)?\.dll$
```

```suggestion
YarnSpinner = (?:^|/)YarnSpinner(?:\.Compiler)?\.dll$
```
not needed since the regex is ran in case insensitive mode

---

```suggestion
YarnSpinner = (?:^|/)YarnSpinner\.dll$
```
after looking into it, the compiler part is also not needed, there no app that have YarnSpinner.Compiler.dll without also having YarnSpinner.dll

---

I've removed the case-sensitivity check, and removed the check for '.Compiler'.

---

Nice, now the only thing left is to fix the test file (remove test with case sensitive & compiler in them)

---

Done!